Python/generated_cases.c.h

```diff
@@ -6027,11 +6027,22 @@
              * marshalling can intern strings and make them immortal. */
             PyObject *obj = GETITEM(FRAME_CO_CONSTS, oparg);
             value = PyStackRef_FromPyObjectNew(obj);
-            #if ENABLE_SPECIALIZATION
+            #if ENABLE_SPECIALIZATION_FT
+            #ifdef Py_GIL_DISABLED
+            uint8_t expected = LOAD_CONST;
+            _Py_atomic_compare_exchange_uint8(
+                &this_instr->op.code, &expected,
+                _Py_IsImmortal(obj) ? LOAD_CONST_IMMORTAL : LOAD_CONST_MORTAL);
```

Comment:
If you want a bigger win, perhaps you could allow deferred references as well. They behave the same as immortal objects if they're live on the stack.

Considering many LOAD_CONST are deferred objects, that should be a nice win in perf.

Reply:
Yeah, that's probably a good idea. Running some tests on the testsuite, about 75% of all LOAD_CONST invocations involved immortal objects, 25% were deferred, and the remainder is less than 1% (in the free-threaded build. Not nearly as many immortal objects in a regular build). I suspect we'll get fewer immortal objects over time, and most of them will probably be deferred instead, I guess?

Now the question is: do I add `PyStackRef_FromPyObjectDeferred()`, or do I change `PyStackRef_FromPyObjectImmortal()` to accept deferred objects as well? :)